packages/fluentui/docs/src/prototypes/mentions/MentionsDropdown.tsx

```diff
@@ -19,6 +19,7 @@ const MentionsDropdown: React.FunctionComponent<MentionsContainerProps> = props
     <Provider.Consumer
       render={({ siteVariables: siteVars }) => (
         <Dropdown
+          highlightFirstItemOnOpen={true}
```

Comment:
you can just pass it as `highlightFirstItemOnOpen` without any parameter.

Reply:
thank you :)